packages/dhis2/src/Adaptor.js

```diff
@@ -146,7 +146,7 @@ axios.interceptors.response.use(
  *   orgUnit: 'DiszpKrYNg8',
  *   status: 'COMPLETED',
  * });
- * @example <caption>a trackedEntityInstance</caption>
+ * @example <caption>a tracker entity. Create a tracker entity instance. See {@link https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-241/tracker.html#webapi_nti_import Create tracker docs}</caption>
```

Comment:
Docs are still weird here. The link is fine but this should continue to say `tracker entity`, so that it's in the same style as the other examples here